src/core/README.md

```diff
@@ -1,4 +1,6 @@
-# Overview
+# gRPC core library
+
+This shared library provides all of gRPC's core functionality through a low
+level API. gRPC libraries for the other languages supported in this repo, are
+built on top of this shared core library.
```

Comment:
Optional: It seems nicer to name each of other languages.

Reply:
My original goal for this PR was to eliminate the list of languages so as to keep things [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Such a change is part of the broader docs work being done in the context of this issue: [_Eliminate duplicate docs across site and repos \#180_](https://github.com/grpc/grpc.io/issues/180).

IMHO, if the languages are to be listed, they should be listed in one place only. The most likely place to have that list is the repo's top-level README (which does contain the list, but it isn't explicitly stated in that way -- I'll see if I can improve the situation when I submit a PR for the top-level README).

Reply:
That make sense 👍